scripts/iso-manager.sh

```diff
@@ -15,34 +24,118 @@ _usage() {
     echo "-d/--dry-run:                    Run actions in dry run mode"
 }
 
-# Exit with error code and message
-die() {
-    echo "$*" 1>&2 ;
-    exit 1;
+PRODUCTS=()
+
+while (( "$#" )); do
+  case "$1" in
+    -p|--product)
+      PRODUCTS+=("$2")
+      shift 2
+      ;;
+    -d|--dry-run)
+      DRY_RUN=True
+      shift
+      ;;
+    -v|--verbose)
+      VERBOSE=1
+      shift
+      ;;
+    -l|--log-file)
+      LOGFILE="$2"
+      [ -z "$DEBUG" ] && DEBUG="on"
+      shift 2
+      ;;
+    *) # unsupported flags
+      echo "Error: Unsupported flag $1" >&2
+      _usage
+      exit 1
+      ;;
+  esac
+done
+
+TMPFILES=$(mktemp -d)
+
+cat << EOF >> "${LOGFILE}"
+--- Bootstrap started on $(date -u -R) ---
```

Comment:
Are we bootstrapping something?

I though that this script was only there to mount ISO and stuff.